.github/workflows/publish_docker.yaml

```diff
@@ -0,0 +1,58 @@
+name: Publish Container to GitHub
```

Comment:
Figure this Action can have other jobs added on for other containers, like `prime-router_builder`.

Reply:
see above, absolutely yes!